run.go

```diff
@@ -328,7 +328,7 @@ func runAction(clicontext *cli.Context) error {
 	portSlice := strutil.DedupeStrSlice(clicontext.StringSlice("p"))
 	netSlice := strutil.DedupeStrSlice(clicontext.StringSlice("net"))
 
-	ports := make([]gocni.PortMapping, len(portSlice))
+	ports := make([]gocni.PortMapping, 0)
```

Comment:
Unneeded change

Reply:
the length of ports slice should be 0 to avoid zero value (nil), nil value in this slice lead nil pointer execption.